### PR TITLE
Update rubocop gems and apply corrections

### PR DESCRIPTION
**Pull request by martinemde**

### CODE REVIEW

**Review by simi — APPROVED** [review]

File: test/models/links_test.rb [hunk 1]
```diff
@@ -62,9 +62,8 @@ class LinksTest < ActiveSupport::TestCase
     rubygem = build(:rubygem, linkset: build(:linkset), versions: [version])
     links = rubygem.links(version)
 
-    enumerated = []
-    links.each do |short, value|
-      enumerated << [short, value]
+    enumerated = links.map do |short, value|
```
Comment (simi) [inline test/models/links_test.rb:65]: this seems wrong, there is only `each` implemented

https://github.com/rubygems/rubygems.org/blob/master/app/models/links.rb#L41-L47

Comment (martinemde) [reply test/models/links_test.rb:65]: Tests caught it. I'm adding Enumerable to Links to fix it.

**Comment by segiddins** [comment]
I'm not seeing an update of the rubocop gems in this diff?

**Comment by martinemde** [comment]
@segiddins oh! I had a rebase problem and had to git chekcout HEAD -- Gemfile.lock and that lost the upgrades after I had already applied the fixes. Funny. Thanks for noticing.

**Comment by martinemde** [comment]
@segiddins fixed.

**Review by segiddins — APPROVED** [review]

**Pull request merged**